homeassistant/components/luftdaten/__init__.py

```diff
@@ -128,39 +101,12 @@ def _async_fixup_sensor_id(hass, config_entry, sensor_id):
     )
 
 
-async def async_setup(hass, config):
-    """Set up the Luftdaten component."""
+async def async_setup_entry(hass, config_entry):
+    """Set up Luftdaten as config entry."""
     hass.data[DOMAIN] = {}
```

Comment:
We can create more than one config entry so we need to use dict.setdefault or similar here.